packages/tools/pie-icons-react/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@justeattakeaway/pie-icons-react",
-  "version": "1.0.0-alpha.0",
-  "tag": "beta",
+  "version": "1.0.0-beta.0",
```

Comment:
The only version was incorrect. Have corrected this to correctly use the `beta` suffix